package.json

```diff
@@ -1,85 +1,85 @@
-    "watch": "run-p watch:*",
-    "watch:main": "node scripts/watch_main_and_preload.js",
-    "watch:renderer": "webpack-dev-server --config webpack.config.renderer.js",
-    "test": "npm-run-all lint:js test:*",
-    "test:app": "cross-env NODE_ENV=production npm run build && mocha -r @babel/register --reporter mocha-circleci-reporter --recursive test/specs",
-    "package:all": "cross-env NODE_ENV=production npm-run-all check-build-config package:windows package:mac package:linux",
-    "package:windows": "cross-env NODE_ENV=production npm-run-all check-build-config build && electron-builder --win --x64 --ia32 --publish=never",
-    "package:mac": "cross-env NODE_ENV=production npm-run-all check-build-config build && electron-builder --mac --publish=never",
-    "package:linux": "cross-env NODE_ENV=production npm-run-all check-build-config build && electron-builder --linux --x64 --ia32 --publish=never",
-    "lint:js": "eslint --ignore-path .gitignore --ignore-pattern node_modules --ext .js --ext .jsx .",
-    "lint:js-quiet": "eslint --ignore-path .gitignore --ignore-pattern node_modules --ext .js --ext .jsx . --quiet",
-    "fix:js": "eslint --ignore-path .gitignore --ignore-pattern node_modules --quiet --ext .js --ext .jsx . --fix",
-    "check-build-config": "node -r @babel/register scripts/check_build_config.js"
-  },
-  "devDependencies": {
-    "7zip-bin": "^4.1.0",
-    "@babel/core": "^7.2.0",
-    "@babel/plugin-proposal-class-properties": "^7.8.3",
-    "@babel/plugin-proposal-object-rest-spread": "^7.2.0",
-    "@babel/preset-env": "^7.2.0",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/register": "^7.0.0",
-    "@storybook/addon-actions": "^4.0.11",
-    "@storybook/react": "^4.0.11",
-    "babel-eslint": "^10.0.3",
-    "babel-loader": "^8.0.4",
-    "chai": "^4.2.0",
-    "cross-env": "^5.2.0",
-    "css-loader": "^1.0.1",
-    "devtron": "^1.4.0",
-    "electron": "^7.2.1",
-    "electron-builder": "^22.2.0",
-    "electron-connect": "^0.6.3",
-    "electron-notarize": "^0.1.1",
-    "eslint": "^6.6.0",
-    "eslint-plugin-cypress": "^2.7.0",
-    "eslint-plugin-eslint-comments": "^3.1.2",
-    "eslint-plugin-header": "^3.0.0",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-mattermost": "github:mattermost/eslint-plugin-mattermost#070ce792d105482ffb2b27cfc0b7e78b3d20acee",
-    "eslint-plugin-react": "^7.16.0",
-    "file-loader": "^2.0.0",
-    "image-webpack-loader": "5.0.0",
-    "mdi-react": "^6.2.0",
-    "mocha": "^5.2.0",
-    "mocha-circleci-reporter": "0.0.3",
-    "npm-run-all": "^4.1.5",
-    "react": "^16.6.3",
-    "react-dom": "^16.6.3",
-    "react-smooth-dnd": "github:mattermost/react-smooth-dnd#af6b471295007274560a375799622c1cd52d678a",
-    "spectron": "^9.0.0",
-    "style-loader": "^0.23.1",
-    "url-loader": "^1.1.2",
-    "webpack": "^4.27.0",
-    "webpack-cli": "^3.1.2",
-    "webpack-dev-server": "^3.1.14",
-    "webpack-merge": "^4.1.4"
-  }
-}
+    "name": "mattermost-desktop",
```

Comment:
I'd suggest getting the original to prevent formatting changes. If I recall the command correctly it should be something like:
```
git checkout origin/master package.json
```

Reply:
done